# Service that can be used to control the robot efforts using the panda_control_server.
# This is done by publishing the joint efforts commands to the right topic.
string[] joint_names
float64[] joint_efforts
bool wait
---
bool success
string message